00-HTML-CSS-basics/css/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+   border-radius: 10px;
+   font-size: 14;
+   font-family: "Times New Roman";
+}
+
+.content {
+    display: inline-block;
+    font-size: 14px;
+    text-align: center;
+    width:46%;
+    vertical-align: top;
+    height: 458px;
+}
+
+.footer {
+    background-color: Magenta;
+    display: block;
+    font-size: 10px ;
+    margin: 0.5% 0.5% 1%;
+    border-radius: 10px;
+    text-align: center;
+    height: 20px;
+}
+
+.header {
+    background-color: Magenta;
+    display: block;
+    font-size: 46px ;
+    margin: 0.5% 0.5% 1%;
+    border-radius: 10px;
+    text-align: center;
+    width:99%;
+}
+
+.navigation {
+    background-color: Blue;
+    display: inline-block;
+    font-size: 12px;
+    margin: 0% 1% 0% 0.5%;
+    vertical-align: top;
+    width:24.8%;
+    height:458px;
+    padding: 0.2%;
+}
+
+.sidebar {
+    background-color: Magenta;
+    display: inline-block;
+    font-size: 10px;
+    margin: 0% 0.5% 0% 1%;
+    vertical-align: top;
+    width:24.8%;
+    padding: 0.2%;
+    height:458px;
+}
+
+.content .header {
+    margin-top: 1%;
+    width: auto;
+    height: 12%;
+}
+
+.content .footer {
+    margin-top: 1%;
+    width: auto;
+    height: 8%;
+}
+
+.content article{
```

Comment:
Not cool using HTML tag as CSS selector. You should use a class for `article` as well.